src/utils/i18n.ts

```diff
@@ -40,3 +40,5 @@
 // t('plugin__kubevirt-plugin~Node')
 // t('plugin__kubevirt-plugin~Create a Virtual Machine from a template')
 // t('plugin__kubevirt-plugin~Hardware Devices')
+// t('plugin__kubevirt-plugin~VMs')
+// t('plugin__kubevirt-plugin~vCPU')
```

Comment:
why do u need it if we can get 't' everywhere we need now? not only as a hook

Reply:
I think it would be a good idea to make a bug for removing this file and adding the function `t` we created wherever needed, but in a different PR

Reply:
Fixed!